internal/clients/benchmark/benchmark.go

```diff
@@ -0,0 +1,136 @@
+package benchmark
+
+import (
+	"SOMAS2023/internal/common/objects"
+	"SOMAS2023/internal/common/physics"
+	"SOMAS2023/internal/common/utils"
+	"math"
+
+	"github.com/google/uuid"
+)
+
+type BenchmarkAgent struct {
+	objects.BaseBiker

```

Comment:
If you want to inherit BaseBiker this should be *objects.BaseBiker

Reply:
sure. i'll add a new subfolder